### DISCUSSION TITLE
10/24 김수진 블로그 생성

### DISCUSSION THREAD

**Discussion by ksj0621** [Orm-camp/blog-feedback#23] [동글동글 3기 피드백]
(블로그 아직 없습니다.)

>일단 블로그를 만들어 운영을 해보는게 좋을 것 같다고 생각했습니다.
>운영자님께서 제공해주신 링크와 다른 훈련생분들의 블로그를 참고하도록 하겠습니다.

**Comment by kimjigyun** [comment]
수진님 말을 많이해주셔서 고마운 분 .. ㅎㅎ 화이팅입니다 ! 응원하고있어요
